jtransc-rt-core/src/com/jtransc/JTranscSystem.java

```diff
@@ -104,7 +104,7 @@ static public void debugger() {
 	@HaxeMethodBody("if (!p0) N.debugger();")
 	@JTranscMethodBodyList({
 		@JTranscMethodBody(target = "js", value = "if (!p0) debugger;"),
-		@JTranscMethodBody(target = "cs", value = "if (!pc) System.Diagnostics.Debugger.Break();"),
+		@JTranscMethodBody(target = "cs", value = "if (!p0) System.Diagnostics.Debugger.Break();"),
```

Comment:
Yes indeed a typo. This one would be harder to test directly. But at least we can detect compile-time errors if referenced.
Maybe we can run a c# test running BigTest but without tree shaking so this kind of methods are still included for compiling even when not executed directly.